README.zh_cn.md

```diff
@@ -18,80 +18,50 @@
-#                                                                       *
-#   根据 R-577 报告规定，本程序将用于指令模块。                            *
-#   本程序为 DSR 项目 55-23870 铺路，（美国）                             *
-#   国家航空航天局的载人航天中心联合 THE INSTRUMENTATION LABORATORY、      *
-#   麻省理工学院、剑桥及MASS签属 NAS 9-4065 合同，共同资助本项目。          *
-#                                                                       *
-#************************************************************************
-
-
-提交者：  MARGARET H. HAMILTON        日期：   1969年3月28日
-    M.H.HAMILTON， COLOSSUS 程序设计负责人
-    阿波罗制导
-
-核准人：   DANIEL J. LICKLY           日期：   1969年3月28日
-    D.J.LICKLY， 负责人、任务程序开发
-    阿波罗制导程序
-
-核准人：   FRED H. MARTIN             日期：   1969年3月28日
-    FRED H. MARTIN， COLOSSUS 项目经理
-    阿波罗制导程序
-
-核准人：   NORMAN E. SEARS            日期：   1969年3月28日
-    N.E. SEARS， 负责人、任务程序开发
-    阿波罗制导程序
-
-核准人：   RICHARD H. BATTIN          日期：   1969年3月28日
-    R.H. BATTIN， 负责人、任务开发
-    阿波罗制导程序
-
-核准人：   DAVID G. HOAG              日期：   1969年3月28日
-    D.G. HOAG， 负责人
-    阿波罗制导程序
-
-核准人：   RALPH R. RAGAN             日期：   1969年3月28日
-    R.R. RAGAN， 副负责人
-    INSTRUMENTATION LABORATORY
-```
 
+&nbsp;      | &nbsp;
+:---------- | :-----
+版权         | 公共领域。
+Comanche055 | Part of the source code for Colossus 2A, the Command Module's (CM) Apollo Guidance Computer (AGC) for Apollo 11<br>`Assemble revision 055 of AGC program Comanche by NASA`<br>`2021113-051. 10:28 APR. 1, 1969`
+Luminary099 | Part of the source code for Luminary 1A, the Lunar Module's (LM) Apollo Guidance Computer (AGC) for Apollo 11<br>`Assemble revision 001 of AGC program LYM99 by NASA`<br>`2021112-061. 16:27 JUL. 14, 1969`
+汇编程序     | yaYUL
+联系人       | Ron Burkey <info@sandroid.org>
+网站         | www.ibiblio.org/apollo
+數字化       | 该源代码从MIT 科学博物馆的印刷本电子化图片抄录、改编而来。电子化工作由 Paul Fjeld 完成，并由该馆的 Deborah Douglas 进一步整理。由衷地感激两位。
+
+### Contract and Approvals
+*Derived from [CONTRACT_AND_APPROVALS.agc]*
+
+本 AGC 程序亦可称为: COLOSSUS 2A
+
+根据 R-577 报告规定，本程序将用于指令模块。本程序为 DSR 项目 55-23870 铺路，（美国）                国家航空航天局的载人航天中心联合 THE INSTRUMENTATION LABORATORY、麻省理工学院、剑桥及MASS签属 NAS 9-4065 合同，共同资助本项目。
+
+提交者                 | Role | 日期
+:-------------------- | :--- | :---
+Margaret H. Hamilton  | Colossus Programming Leader<br>Apollo Guidance and Navigation |  1969年3月28日
+
+核准人              | Role | 日期
+:----------------- | :--- | :---
+Daniel J. Lickly   | Director, Mission Program Development<br>Apollo Guidance and Navigation Program | 1969年3月28日
+Fred H. Martin     | Colossus Project Manager<br>Apollo Guidance and Navigation Program |  1969年3月28日
+Norman E. Sears    | Director, Mission Development<br>Apollo Guidance and Navigation Program | 1969年3月28日
+Richard H. Battin  | Director, Mission Development<br>Apollo Guidance and Navigation Program | 1969年3月28日
+David G. Hoag      | Director<br>Apollo Guidance and Navigation Program | 1969年3月28日
+Ralph R. Ragan     | Deputy Director<br>Instrumentation Laboratory | 1969年3月28日
```

Comment:
These fields were already localized in the target language, but they were pasted in English.

Reply:
Hmm looks so.